### PR TITLE
[stable/parse] New major version]

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juan@bitnami.com>

#### What this PR does / why we need it:

This PR bumps the major version of the chart to include some breaking changes:

- Labels are adapted to follow the Helm charts best practices
- **parse-dashboard** is bumped to the branch 2 (major version)

It also adds the chart support for some common features such as custom liveness/readiness probes, tolerations, and nodeSelector.

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/parse/templates/NOTES.txt [hunk 1]
```diff
@@ -78,8 +79,9 @@ service:
 
 {{- else if eq .Values.service.type "ClusterIP" }}
 
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/name={{ template "parse.name" . }},app.kubernetes.io/component=dashboard -o jsonpath="{.items[0].metadata.name}")
```
Comment (miguelaeh) [inline stable/parse/templates/NOTES.txt:82]: The port-forward should be done to a service. Also, we should avoid to hardcode the ports.

File: stable/parse/templates/NOTES.txt [hunk 2]
```diff
@@ -18,20 +18,21 @@ Parse Server
...
-        Watch the status with: 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "parse.fullname" . }}'
+        Watch the status with: 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ include "parse.fullname" . }}'
 
-  export SERVICE_HOST=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "parse.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}"):{{ .Values.server.port }}
+  export SERVICE_HOST=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "parse.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}"):{{ .Values.server.port }}
 
 {{- else if contains "ClusterIP"  .Values.service.type }}
 
-  kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ template "parse.fullname" . }} 1337:{{ .Values.server.port }} &
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/name={{ template "parse.name" . }},app.kubernetes.io/component=server -o jsonpath="{.items[0].metadata.name}")
```
Comment (miguelaeh) [inline stable/parse/templates/NOTES.txt:34]: The port-forward should be done to a service. Also, we should avoid to hardcode the ports as the service has a configurable port.

File: stable/parse/templates/NOTES.txt [hunk 3]
```diff
@@ -90,7 +92,7 @@ service:
 2. Get your Parsh Dashboard login credentials by running:
```
Comment (miguelaeh) [inline stable/parse/templates/NOTES.txt]:
```diff
- Parsh
+ Parse
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you for the PR!!!!
Please review the comments.

File: stable/parse/templates/NOTES.txt (hunk 1 above)
Comment (juan131) [reply stable/parse/templates/NOTES.txt:82]: This chart is a little special. It's using a single service for both "server" and "dashboard" pods (playing with the common labels). 

This peculiarity is incompatible with `port-forwarding` to services.

File: stable/parse/templates/NOTES.txt (hunk 2 above)
Comment (juan131) [reply stable/parse/templates/NOTES.txt:34]: Idem

**Pull request merged**